<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Magento Web API Functional Tests" tests="28" assertions="0" errors="28" failures="0" skipped="0" time="9.717228">
    <testsuite name="Magento\Webpos\Api\Cart\CartTaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTaxClassTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="5.153336">
      <testcase name="testGetList" class="Magento\Webpos\Api\Cart\CartTaxClassTest" classname="Magento.Webpos.Api.Cart.CartTaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTaxClassTest.php" line="24" assertions="0" time="5.153336">
        <error type="Exception">Magento\Webpos\Api\Cart\CartTaxClassTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1024675567393&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTaxClassTest.php:40
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Cart\CartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="1.519253">
      <testcase name="testSendEmail" class="Magento\Webpos\Api\Cart\CartTest" classname="Magento.Webpos.Api.Cart.CartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTest.php" line="24" assertions="0" time="1.519253">
        <error type="Exception">Magento\Webpos\Api\Cart\CartTest::testSendEmail
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1108665369218&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CartTest.php:36
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Catalog\SwatchRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.126567">
      <testcase name="testGetList" class="Magento\Webpos\Api\Catalog\SwatchRepositoryTest" classname="Magento.Webpos.Api.Catalog.SwatchRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php" line="35" assertions="0" time="0.126567">
        <error type="Exception">Magento\Webpos\Api\Catalog\SwatchRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 676499667569&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php:51
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Categories\SearchCategoriesCriteriaTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Categories/SearchCategoriesCriteriaTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.117426">
      <testcase name="testGetListCategories" class="Magento\Webpos\Api\Categories\SearchCategoriesCriteriaTest" classname="Magento.Webpos.Api.Categories.SearchCategoriesCriteriaTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Categories/SearchCategoriesCriteriaTest.php" line="59" assertions="0" time="0.117426">
        <error type="Exception">Magento\Webpos\Api\Categories\SearchCategoriesCriteriaTest::testGetListCategories
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 925414895305&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Categories/SearchCategoriesCriteriaTest.php:45
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Categories/SearchCategoriesCriteriaTest.php:60
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\PlaceOrderTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/PlaceOrderTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.232011">
      <testcase name="testPlaceOrder" class="Magento\Webpos\Api\Checkout\PlaceOrderTest" classname="Magento.Webpos.Api.Checkout.PlaceOrderTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/PlaceOrderTest.php" line="190" assertions="0" time="0.116497">
        <error type="Exception">Magento\Webpos\Api\Checkout\PlaceOrderTest::testPlaceOrder
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 684135104240&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/PlaceOrderTest.php:78
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/PlaceOrderTest.php:192
</error>
      </testcase>
      <testcase name="testMultiPlaceOrder" class="Magento\Webpos\Api\Checkout\PlaceOrderTest" classname="Magento.Webpos.Api.Checkout.PlaceOrderTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/PlaceOrderTest.php" line="380" assertions="0" time="0.115514">
        <error type="Exception">Magento\Webpos\Api\Checkout\PlaceOrderTest::testMultiPlaceOrder
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 813937523673&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/PlaceOrderTest.php:382
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\RemoveItemTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/RemoveItemTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.115228">
      <testcase name="testRemoveItem" class="Magento\Webpos\Api\Checkout\RemoveItemTest" classname="Magento.Webpos.Api.Checkout.RemoveItemTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/RemoveItemTest.php" line="100" assertions="0" time="0.115228">
        <error type="Exception">Magento\Webpos\Api\Checkout\RemoveItemTest::testRemoveItem
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 871966840711&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/RemoveItemTest.php:83
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/RemoveItemTest.php:102
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\SaveCartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveCartTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.114959">
      <testcase name="testSaveCart" class="Magento\Webpos\Api\Checkout\SaveCartTest" classname="Magento.Webpos.Api.Checkout.SaveCartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveCartTest.php" line="293" assertions="0" time="0.114959">
        <error type="Exception">Magento\Webpos\Api\Checkout\SaveCartTest::testSaveCart
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1323984677809&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveCartTest.php:58
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveCartTest.php:294
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\SaveQuoteDataTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveQuoteDataTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.115883">
      <testcase name="testSaveQuoteData" class="Magento\Webpos\Api\Checkout\SaveQuoteDataTest" classname="Magento.Webpos.Api.Checkout.SaveQuoteDataTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveQuoteDataTest.php" line="79" assertions="0" time="0.115883">
        <error type="Exception">Magento\Webpos\Api\Checkout\SaveQuoteDataTest::testSaveQuoteData
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 594036954740&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveQuoteDataTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveQuoteDataTest.php:81
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\SaveShippingMethodTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveShippingMethodTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.116222">
      <testcase name="testSaveShippingMethod" class="Magento\Webpos\Api\Checkout\SaveShippingMethodTest" classname="Magento.Webpos.Api.Checkout.SaveShippingMethodTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveShippingMethodTest.php" line="88" assertions="0" time="0.116222">
        <error type="Exception">Magento\Webpos\Api\Checkout\SaveShippingMethodTest::testSaveShippingMethod
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 506992979458&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveShippingMethodTest.php:62
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SaveShippingMethodTest.php:90
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\SendEmailTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SendEmailTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.113455">
      <testcase name="testSendEmail" class="Magento\Webpos\Api\Checkout\SendEmailTest" classname="Magento.Webpos.Api.Checkout.SendEmailTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SendEmailTest.php" line="39" assertions="0" time="0.113455">
        <error type="Exception">Magento\Webpos\Api\Checkout\SendEmailTest::testSendEmail
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 755908207457&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/SendEmailTest.php:40
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Config\ConfigRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.113974">
      <testcase name="testGetList" class="Magento\Webpos\Api\Config\ConfigRepositoryTest" classname="Magento.Webpos.Api.Config.ConfigRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php" line="24" assertions="0" time="0.113974">
        <error type="Exception">Magento\Webpos\Api\Config\ConfigRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 865858492330&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php:33
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Customer\CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" tests="3" assertions="0" errors="3" failures="0" skipped="0" time="0.348149">
      <testcase name="testGetList" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="49" assertions="0" time="0.118629">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1189600997611&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:71
</error>
      </testcase>
      <testcase name="testSave" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="94" assertions="0" time="0.115724">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testSave
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 426057353498&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:114
</error>
      </testcase>
      <testcase name="testSearchCustomer" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="135" assertions="0" time="0.113796">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testSearchCustomer
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 368028036579&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:136
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\GetListSession\GetListSessionTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/GetListSession/GetListSessionTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.113326">
      <testcase name="testGetListShiftSession" class="Magento\Webpos\Api\GetListSession\GetListSessionTest" classname="Magento.Webpos.Api.GetListSession.GetListSessionTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/GetListSession/GetListSessionTest.php" line="81" assertions="0" time="0.113326">
        <error type="Exception">Magento\Webpos\Api\GetListSession\GetListSessionTest::testGetListShiftSession
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1029256832768&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/GetListSession/GetListSessionTest.php:45
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/GetListSession/GetListSessionTest.php:82
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Intergration\SpendPointTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Intergration/SpendPointTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.116490">
      <testcase name="testSpendPoint" class="Magento\Webpos\Api\Intergration\SpendPointTest" classname="Magento.Webpos.Api.Intergration.SpendPointTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Intergration/SpendPointTest.php" line="49" assertions="0" time="0.116490">
        <error type="Exception">Magento\Webpos\Api\Intergration\SpendPointTest::testSpendPoint
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 734528986123&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Intergration/SpendPointTest.php:51
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Inventory\StockItemRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.116979">
      <testcase name="testGetStockItems" class="Magento\Webpos\Api\Inventory\StockItemRepositoryTest" classname="Magento.Webpos.Api.Inventory.StockItemRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php" line="24" assertions="0" time="0.116979">
        <error type="Exception">Magento\Webpos\Api\Inventory\StockItemRepositoryTest::testGetStockItems
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1360634774950&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php:47
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Payment\PaymentRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.112989">
      <testcase name="testGetList" class="Magento\Webpos\Api\Payment\PaymentRepositoryTest" classname="Magento.Webpos.Api.Payment.PaymentRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php" line="35" assertions="0" time="0.112989">
        <error type="Exception">Magento\Webpos\Api\Payment\PaymentRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 829208398342&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php:43
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Pos\PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.115522">
      <testcase name="testAssignStaff" class="Magento\Webpos\Api\Pos\PosRepositoryTest" classname="Magento.Webpos.Api.Pos.PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" line="44" assertions="0" time="0.115522">
        <error type="Exception">Magento\Webpos\Api\Pos\PosRepositoryTest::testAssignStaff
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 936104508787&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php:54
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\PosAssign\AssignPosTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PosAssign/AssignPosTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.114198">
      <testcase name="testAssignStaff" class="Magento\Webpos\Api\PosAssign\AssignPosTest" classname="Magento.Webpos.Api.PosAssign.AssignPosTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PosAssign/AssignPosTest.php" line="35" assertions="0" time="0.114198">
        <error type="Exception">Magento\Webpos\Api\PosAssign\AssignPosTest::testAssignStaff
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1012458873466&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PosAssign/AssignPosTest.php:37
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\PostList\PostListTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PostList/PostListTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.129686">
      <testcase name="testPostListSearchCriteria" class="Magento\Webpos\Api\PostList\PostListTest" classname="Magento.Webpos.Api.PostList.PostListTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PostList/PostListTest.php" line="50" assertions="0" time="0.129686">
        <error type="Exception">Magento\Webpos\Api\PostList\PostListTest::testPostListSearchCriteria
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1113246634829&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PostList/PostListTest.php:37
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/PostList/PostListTest.php:51
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\ProductList\ProductListOutOfStockTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/ProductList/ProductListOutOfStockTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.128985">
      <testcase name="testGetListProductsOutOfStock" class="Magento\Webpos\Api\ProductList\ProductListOutOfStockTest" classname="Magento.Webpos.Api.ProductList.ProductListOutOfStockTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/ProductList/ProductListOutOfStockTest.php" line="59" assertions="0" time="0.128985">
        <error type="Exception">Magento\Webpos\Api\ProductList\ProductListOutOfStockTest::testGetListProductsOutOfStock
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1186546825000&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/ProductList/ProductListOutOfStockTest.php:45
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/ProductList/ProductListOutOfStockTest.php:60
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Sales\OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.124293">
      <testcase name="testGetList" class="Magento\Webpos\Api\Sales\OrderRepositoryTest" classname="Magento.Webpos.Api.Sales.OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" line="35" assertions="0" time="0.124293">
        <error type="Exception">Magento\Webpos\Api\Sales\OrderRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1464476712208&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php:57
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Shifts\GetListShiftTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shifts/GetListShiftTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.116118">
      <testcase name="testGetListShift" class="Magento\Webpos\Api\Shifts\GetListShiftTest" classname="Magento.Webpos.Api.Shifts.GetListShiftTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shifts/GetListShiftTest.php" line="51" assertions="0" time="0.116118">
        <error type="Exception">Magento\Webpos\Api\Shifts\GetListShiftTest::testGetListShift
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 638322487810&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shifts/GetListShiftTest.php:38
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shifts/GetListShiftTest.php:52
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Staff\LoginTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.114292">
      <testcase name="testStaffLogin" class="Magento\Webpos\Api\Staff\LoginTest" classname="Magento.Webpos.Api.Staff.LoginTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php" line="36" assertions="0" time="0.114292">
        <error type="Exception">Magento\Webpos\Api\Staff\LoginTest::testStaffLogin
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 517682591879&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Staff\StaffManagementTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.113422">
      <testcase name="testStaffLogin" class="Magento\Webpos\Api\Staff\StaffManagementTest" classname="Magento.Webpos.Api.Staff.StaffManagementTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php" line="31" assertions="0" time="0.113422">
        <error type="Exception">Magento\Webpos\Api\Staff\StaffManagementTest::testStaffLogin
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 447436576538&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php:33
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\TaxClass\TaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/TaxClass/TaxClassTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.114465">
      <testcase name="testSearchCriteria" class="Magento\Webpos\Api\TaxClass\TaxClassTest" classname="Magento.Webpos.Api.TaxClass.TaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/TaxClass/TaxClassTest.php" line="34" assertions="0" time="0.114465">
        <error type="Exception">Magento\Webpos\Api\TaxClass\TaxClassTest::testSearchCriteria
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1241521968564&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/LoginTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/TaxClass/TaxClassTest.php:36
</error>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
